### PR TITLE
Adding a notification when user leave meet room.

### PR DESCRIPTION
**Pull request by flucas97**
fix:  #24

obs: I could not test on my local enviroment, please check before merge.

### CODE REVIEW

File: public/js/office.web.js [hunk 1]
```diff
@@ -264,7 +275,11 @@ $(() => {
     initGetUserMenu(officeEvents);
 
     officeEvents.onParticipantStartedMeet(showUserInRoom);
-    officeEvents.onParticipantLeftMeet(showUserInRoom);
+    officeEvents.onParticipantLeftMeet(function(user,roomId){
```
Comment (lccezinha) [inline public/js/office.web.js]:
```suggestion
    officeEvents.onParticipantLeftMeet(function(user ,roomId) {
```

File: public/js/office.web.js [hunk 2]
```diff
@@ -264,7 +275,11 @@ $(() => {
...
+      showUserInRoom(user,roomId);
```
Comment (lccezinha) [inline public/js/office.web.js]:
```suggestion
      showUserInRoom(user, roomId);
```

File: public/js/office.web.js [hunk 3]
```diff
@@ -264,7 +275,11 @@ $(() => {
...
+      notifyRoomExited(user,roomId);
```
Comment (lccezinha) [inline public/js/office.web.js]:
```suggestion
      notifyRoomExited(user, roomId);
```

File: public/js/office.web.js (hunk 1 above)
Comment (lccezinha) [inline public/js/office.web.js]: To keep the same approach that we have in the `officeEvents.onParticipantStartedMeet` function, maybe we could create another function called

- `showUserLeaveRoom`

```javascript
function showUserLeaveRoom(user, roomID) {
  showUserInRoom(user,roomId);
  notifyRoomExited(user,roomId);
}
```

and use like this:

```javascript
officeEvents.onParticipantStartedMeet(showUserInRoom);
officeEvents.onParticipantLeftMeet(showUserLeaveRoom);
```

What do you think?

**Comment by juliemar** [comment]
Hi @flucas97 are you still working in this solution?

**Comment by lccezinha** [comment]
@juliemar Do you know how can we push to this branch? I tried to fork into my local and do the fixes, but I was not able, because the remote is `flucas97` and I do not know how to handle it.

**Comment by flucas97** [comment]
Good night Juliemar and Luiz,

I'm sorry for the delay in answering, I've been a few days without seeing my github.

I would like to be able to apply the suggestions for changes in the code, I am in the process of rotation for engineering, Monday was my first day. I ended up getting stuck in ramping tasks and have not taken the time to figure out the hint yet.

If you want to merge, its ok apply the suggestion, its not a problem for me.

**Comment by lccezinha** [comment]
@flucas97 We could do a pair programming in the next week if you want to solve this problem. I think 30 minutes will be enough.

**Comment by flucas97** [comment]
@lccezinha I would like it!, when works best for you?

**Comment by juliemar** [comment]
I'm closing this PR. don't have activity for a long time.

**Pull request closed without merging**